.github/actions/chocolatey/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine:latest
+
+ARG CHOCOVERSION=1.1.0
+
+RUN apk add --no-cache bash ca-certificates git \
+  && apk --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing add mono mono-dev \
+  && cert-sync /etc/ssl/certs/ca-certificates.crt \
+  && wget "https://github.com/chocolatey/choco/archive/${CHOCOVERSION}.tar.gz" -O- | tar -xzf - \
+  && cd choco-"${CHOCOVERSION}" \
+  && chmod +x build.sh zip.sh \
+  && ./build.sh -v \
+  && mv ./code_drop/chocolatey/console /opt/chocolatey \
```

Comment:
@ghuntley these steps shouldn't be necessary.  Manually building Chocolatey is no longer required, as we ship a Docker Image ready to be used: https://hub.docker.com/u/chocolatey